cmd/podman/container.go

```diff
@@ -54,6 +54,7 @@ var (
 		_checkpointCommand,
 		_containerExistsCommand,
 		_contInspectSubCommand,
+		_cpCommand,
```

Comment:
@baude does this need to be added to mainCommands in cmd/podman/main.go too?

Reply:
This confused me a bit, since the command is documented and there, but apparently not in the main?